src/lib/viewers/archive/Breadcrumbs.js

```diff
@@ -19,14 +19,21 @@ class Breadcrumbs extends React.PureComponent {
      * @return {Array<Object>} path items including name and path string
      */
     getPathItems = fullPath => {
+        if (fullPath === ROOT_FOLDER) {
+            return [{ name: __('root_folder'), path: ROOT_FOLDER }];
+        }
+
         const pathNames = fullPath.split('/').slice(0, -1);
         // join path names from root to current index to get absolute path
         const getAbsolutePath = index => pathNames.slice(0, index + 1).join('/');
 
-        return pathNames.map((name, index) => ({
-            name,
-            path: `${getAbsolutePath(index)}/`,
-        }));
+        return [
+            { name: __('root_folder'), path: ROOT_FOLDER },
```

Comment:
Is this needed if the zip file already has a root folder? If so, can we avoid duplicating this object declaration by pushing the other pathnames onto a `results` array?

Reply:
> Is this needed if the zip file already has a root folder? If so, can we avoid duplicating this object declaration by pushing the other pathnames onto a `results` array?

I researched G Drive and observed they add root folder for all zip files. So I think this is needed if the zip file already has a root folder.

We cannot prepend it the the results array because the filepath is 'root/' not empty string, so it will add another 'root' object.